cves/kernel/CVE-2021-23133.yml

```diff
@@ -313,8 +332,11 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
+  discussed_as_security: false
+  any_discussion: | 
+    There was some discussion between two developers regarding whether a previous
+    commit that introduced the bug would be the proper fix or pushing another fixed
```

Comment:
Wording of this section is a bit weird. Breaking it up into multiple sentences might help improve the clarity. Something like: 
There was some discussion between two developers around how to fix the vulnerability. One developer argued that ...
The other developer argued ... would be a better solution. Ultimately, the solution was ...